### ISSUE TITLE
Empty functions: only abort() calls in function bodies

### ISSUE THREAD

**Issue by PeterMatula** [avast-tl/retdec#376]
Some functions from [sample.zip](https://github.com/avast-tl/retdec/files/2293569/sample.zip) (probably Delphi) decompile (via RetDec IDA plugin) into functions with empty bodies with only `abort()` call and `UNREACHABLE` comment. E.g.

```c
// Address range: 0x4169830c - 0x4169837c
int32_t sub_4169830C(void) {
    // 0x4169830c
    abort();
    // UNREACHABLE
}
```

```c
int32_t TModuleEntry(int32_t a1, int32_t a2, int32_t a3, int32_t a4) {
    // 0x4169798c
    abort();
    // UNREACHABLE
}
```

**Comment by palant** [comment]
This is a duplicate of #169 I'd say.

**Comment by PeterMatula** [comment]
Fixed by merging #391.
